src/inc/common/Mlp_Helpers.php

```diff
@@ -479,7 +479,7 @@ public static function show_linked_elements( $args ) {
 
 		$output .= '</ul></div>';
 
-		return $output;
+		return apply_filters( 'show_linked_elements', $output, $items, $params );
```

Comment:
In my opinon `mlp_linked_elements_html` would better fit the existing [filters](../search?p=3&q=apply_filters%28&utf8=%E2%9C%93).